src/docker-osx-dev

```diff
@@ -846,6 +853,21 @@ function initial_sync {
   log_debug "Creating parent directories in Docker VM: $ssh_cmd"
   $DOCKER_HOST_SSH_COMMAND "$ssh_cmd"
 
+  local readonly excludes=($EXCLUDES)
+  exclude_flags=${excludes[@]/#/--exclude }
+  local readonly includes=($INCLUDES)
+  include_flags=${includes[@]/#/--include }
+  for key in "${!parent_dirs[@]}"; do
+    parent_dir=${parent_dirs[$key]}
+    base_name=${base_names[$key]}
+    if $DOCKER_HOST_SSH_COMMAND "test -e '$parent_dir/$base_name'" > /dev/null 2>&1; then
+      log_debug "skipped tar for $parent_dir/$base_name"
+    else
+      log_info "Initial sync using tar for $parent_dir/$base_name"
+      tar -cC "$parent_dir" $include_flags $exclude_flags "$base_name" | $DOCKER_HOST_SSH_COMMAND "tar -xC '$parent_dir'"
```

Comment:
Just to make sure I understand this, are you piping the output of the first `tar` command into the input of the second `tar` command executed via `ssh`? If so, that's a clever approach. How does that compare, performance wise, to just `rsync`ing or `scp`ing the tar file?